packages/create-bison-app/template/package.json.ejs

```diff
@@ -6,14 +6,15 @@
   "cacheDirectories": [".next/cache"],
 <% } -%>
   "scripts": {
-    "build": "ts-node ./scripts/buildProd",
+    "build": "node ./scripts/buildProd",
     "build:prisma": "prisma generate",
     "build:next": "next build",
     "build:nexus": "NODE_ENV=development ts-node -P tsconfig.cjs.json --transpile-only graphql/schema.ts",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "db:migrate": "yarn -s prisma migrate up --experimental && yarn build:prisma",
-    "db:rollback": "yarn prisma migrate down --experimental && yarn build:prisma",
+    "db:migrate": "yarn -s prisma migrate dev && yarn build:prisma",
+    "db:deploy": "yarn -s primsa deploy && yarn build:prisma",
+    "db:reset": "yarn prisma migrate reset && yarn build:prisma",
```

Comment:
Prisma no longer offers a direct rollback feature or `migrate down` so we renamed the script to match the functionality which resets the database and then runs the seed file if found. This is intended to be used on dev only, not on production database.